### DISCUSSION TITLE
How  to not interpolate in the blank without point cloud when build DTM？

### DISCUSSION THREAD

**Discussion by QAQ0w0** [r-lidar/lidR#575] [Q&A]
Because the shape of my file is irregular, there are some prominent ground points.When build DTM with lidR，points will be generated by interpolation in some places that do not belong to my work area.
such as this picture，blue area is my work area，and red area before dtm is blank ，but after building dtm this area become a ground in dtm
![无标题](https://user-images.githubusercontent.com/104728433/166155324-6205bf9f-0b03-40ac-9f24-ce3c08e6bb37.png)

**Comment by Jean-Romain** [comment]
There are multiple solutions. `tin()` has a parameter `max_edge`. `rasterize_terrain` has generic argument `shape`

**Reply by QAQ0w0** [reply]
Is this feature not available in version 3.2.0? I haven't upgraded to version 4.0.0 yet

**Reply by Jean-Romain** [reply]
Yes but not exactly implemented the same way.  Anyway, please update
